website/search/elastic_search.py

```diff
@@ -418,11 +416,10 @@ def serialize_contributors(node):
     return {
         'contributors': [
             {
-                'fullname': user.fullname,
-                'url': user.profile_url if user.is_active else None
-            } for user in node.visible_contributors
-            if user is not None
-            and user.is_active
+                'fullname': x['user__fullname'],
+                'url': '/{}/'.format(x['user__guids___id'])
+            } for x in
+            node.contributor_set.filter(visible=True, user__is_active=True).order_by('contributor___order').values('user__fullname', 'user__guids___id')
```

Comment:
I would imagine that doing the extra queries would be faster than filtering in python in almost all circumstances. Regardless, we should fix include, or stop using it, if this doesn't work. Chaining is a very highly used feature of the ORM.